src/master/concepts.md

```diff
@@ -1,109 +1,50 @@
-
-<WrappedSection>
-
-## Checking data consistency
-This section is not yet ready for immudb 0.9. We are working on it in order to improve it and we are close to deliver. Stay tuned!
-
-</WrappedSection>
-
-<WrappedSection>
-
-## State signature
-
-Providing `immudb` with a signing key enables the cryptographic state signature.
-That means that an auditor or a third party client, for instance, could verify the authenticity of the returned current state after calling the `currentState` gRPC method.
-
-Here are the gRPC message definitions:
-```
-message ImmutableState {
-	uint64 txId = 3;
-	bytes txHash = 4;
-	Signature signature = 5;
-}
-
-message Signature {
-	bytes signature = 1;
-	bytes publicKey = 2;
-}
-```
-
-Check [state signature](old/immudb/#state-signature) and [verify state signature](sdks-api.html#verify-state-signature) paragraphs for additional details.
+To increase security, immudb can be provisioned with a signing key to ensure non-repudiation of database states. The immudb server will subsequently sign the state on request from a client application.
+That means that an auditor or a third party client, for instance, could verify the authenticity of the returned current state.
 
 Immuclient and [immugw](https://github.com/codenotary/immugw) are shipped with auditor capabilities.
-To get the signed state in combination with the auditor, launch...
-* ...immuclient with auditor capabilities:
+To get the (signed) state in combination with the immuclient with auditor capabilities:
 ```bash
-immuclient audit-mode --audit-username {immudb-username} --audit-password {immudb-pw} --audit-signature validate
+immuclient audit-mode --audit-username {immudb-username} --audit-password {immudb-pw} --server-signing-pub-key {state-public-key}
 ```
-* ...with [immugw](https://github.com/codenotary/immugw) with auditor capabilities:
-```bash
-./immugw --audit --audit-username {immudb-username} --audit-password {immudb-pw} --audit-signature validate
-```
-
-</WrappedSection>
 
-<WrappedSection>
-
-## Item References
-
-Enables the insertion of a special entry that references another item.
-
-</WrappedSection>
-
-<WrappedSection>
-
-## Primary Index
-
-The primary index enables queries and search based on the **data key**.
+Check [tamper-proof operations](develop/operations.htm) and [running an auditor with immuclient](develop/auditor.html) paragraphs for additional details.
 
 </WrappedSection>
 
 <WrappedSection>
 
-## Secondary Index
+## Key value and SQL
 
-The secondary index enables queries and search based on the **data value**.
-
-</WrappedSection>
-
-<WrappedSection>
-
-## Streams
-Allows client server communication with streams of “delimited” []byte messages.
-
-</WrappedSection>
-
-<WrappedSection>
+Immudb can be used as a tamper-proof key value store or SQL database, with audit history capabilities. Within single immudb instance a user can have both types of databases, it is even possible to have KV and SQL withing single database.
```

Comment:
I think that this is still missing explanation that a single immudb instance can contain multiple databases.

Reply:
refactored a little bit, should be quite clear now